src/context-providers/builder/element.types.ts

```diff
@@ -82,6 +86,9 @@ export interface ITextareaFieldAttributes
     extends IBaseTextBasedFieldAttributes {
     resizableInput?: boolean;
     preSelectedValue?: string;
+    pills?: boolean;
+    pillItems?: IPillItemAttributes[];
```

Comment:
why do `pillItems` need to be an array of objects? wont having an array of strings be easier?

Reply:
This is because `pillItems` is implemented with `useFieldArray` and it requires the fields to be objects. 

From the [docs](https://react-hook-form.com/docs/usefieldarray),

`fields` has type `object & { id: string }`